commit 26b0feb207919293098a1eba70d41d324642a4f5 (refs/changes/16/516/4)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-23T18:31:07+00:00 (3 years ago)
    
    IMGAPI-571 re-do images.jo/updates.jo deployment to be able use stock 'imgapi' images
    IMGAPI-567 Update stud for OpenSSL 1.0.1t/1.0.2h vulnerability fixes
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/build.spec b/build.spec
index 157abea6..f3cfae13 100644
--- a/build.spec
+++ b/build.spec
@@ -136,6 +136,12 @@
       "name": "sdc-base",
       "version": "14.2.0",
       "uuid": "de411e86-548d-11e4-a4b7-3bb60478632a"
+    },
+    "sdc-minimal-multiarch-lts-15.4.1": {
+      "imgapi": "https://updates.joyent.com",
+      "name": "sdc-minimal-multiarch-lts",
+      "version": "15.4.1",
+      "uuid": "18b094b0-eb01-11e5-80c1-175dac7ddf02"
     }
   }
 }
diff --git a/config/sapi/services/imgapi/service.json b/config/sapi/services/imgapi/service.json
index 3a202415..62d0d344 100644
--- a/config/sapi/services/imgapi/service.json
+++ b/config/sapi/services/imgapi/service.json
@@ -15,7 +15,6 @@
     },
     "metadata": {
         "SERVICE_NAME": "imgapi",
-        "STORAGE_LOCAL_BASEDIR": "/data/imgapi",
         "STORAGE_MANTA_BASEDIR": "imgapi"
     },
     "manifests": {
